
---
float64 x
float64 y
float64 z
float64 ox
float64 oy
float64 oz
float64 ow
float64 a
float64 b
float64 c
float64 d
float64 e
float64 f